queue_job_context/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Queue Job, prepare context before enqueue keys
```

Comment:
Please mention the keys that are propagated in the job context here, and also mention that it will become the default in 16.0.

Reply:
done!